### PR TITLE
I blocked Andy

### PR DESCRIPTION
**Pull request by BenEmdon**

### CODE REVIEW

File: _pages/leaderboard.md [hunk 1]
```diff
@@ -5,10 +5,10 @@ permalink: /leaderboard
 
 | Name          | Iced          | Got Iced   |
 | ------------- |:-------------:|:----------:|
-| Andy          | 12            | 2          |
+| Andy          | 12            | 3          |
 | Sam           | 3             | 2          |
 | Jacky         | 2             | 7          |
-| Ben           | 1             | 2          |
+| Ben           | 2             | 2          |
```
Comment (JackyChiu) [inline _pages/leaderboard.md]: @BenEmdon wait I think you deserve to get +2 here because he had to drink both

Comment (samjjc) [reply _pages/leaderboard.md]: I don't think andy actually drank the litre

Comment (JackyChiu) [reply _pages/leaderboard.md]: Oh wait nvm. Andy asked Ben to be blocked with an regular ice instead, that liter was from Bernard. All good here.

Comment (BenEmdon) [reply _pages/leaderboard.md]: Did he drink both tho?

**Pull request merged**